### PR TITLE
Sticky ad 4

### PR DESCRIPTION
**Pull request by VladDubrovskis**
**Changes:**

1. Removed all the right hand rail stickiness just to keep it simple
2. The sticky ads inits once the ad is being loaded on the page, that way we let it expand first. We could test if this works for now and see if it actually has got any extra view ability impact. And if its too low could improve it then?
3. IE is not ideal and sometimes still hover the menu a little but only for a split second.

Very similar to the original solution, just tidied up the event listeners and references

/cc @adgad @andrewgeorgiou1981 @gvonkoss

### CODE REVIEW

File: ads/js/sticky.js [hunk 1]
```diff
@@ -1,111 +1,125 @@
...
 
 function Sticky (el, opts) {
-	if (!el) return;
-	this.el = el;
 	this.opts = opts || {};
+	this.fixed = el;
+	this.boundary = document.querySelector(this.opts.stickUntil);
+	this.eventdbScrollEnd = debounce(this.scrollEnd.bind(this), 300);
+	this.eventScrollStart = this.scrollStart.bind(this);
 	this.sibling = opts.sibling ? document.querySelector(opts.sibling) : null;
```
Comment (gvonkoss) [inline ads/js/sticky.js]: this was specific to RHR, the stickey header technically needs a sibling. I think.

**Comment by andrewgeorgiou1981** [comment]
👍

File: ads/js/sticky.js [hunk 2]
```diff
@@ -1,111 +1,119 @@
...
-	} else if (!this.onScrollListener && this.el.offsetHeight >= 10) {
-		this.bindScroll();
+	});
+}
+
+Sticky.prototype.calculate = function () {
+	const scrollY = window.pageYOffset || window.scrollY;
+	const atBoundary = (scrollY - this.startScroll + this.fixedHeight) >= this.boundaryTop;
+	const isAbsolute = this.fixed.style.position === 'absolute';
+	const extraAbsolute = ((scrollY - this.extraHeight) >= 0);
```
Comment (adgad) [inline ads/js/sticky.js]: Perhaps a more explanatory variable name here?

**Comment by adgad** [comment]
I think (separate to this PR) we should add a class where we want it sticky, and only do it there. i.e. the myFT sponsorship probably shouldn't be sticky.

**Pull request merged**